### PR TITLE
Initial structure and readme files for repo

### PR DESCRIPTION
**Pull request by smit1678**
This is based on the ideas I put down for how we structure this repo in https://github.com/hotosm/tech/issues/96. 

I've added some initial versions of the readme files plus updated the main README.

### CODE REVIEW

File: quarterly-goals/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Quarterly Goals
+
+Quarterly goals is a place where the community can document direction and communicate focus on a quarterly basis. Openness is a key part of our mission and we want to share where and how we operate and invite others to participate. 
+
+### Why have quarterly goals? 
+
+Quarterly goals gives focus on the short-term. Setting bigger picture one or five year goals can be helpful but overwhelming. By setting smaller, achievable goals on a three-month basis, we can create focus and better monitor our progress. 
+
+Setting quarterly goals also achieves two critical items in our community: 1) agreeing upon direction in smaller pieces; and 2) help us internally evaluate to improve how we communicate.
```
Comment (ericboucher) [inline quarterly-goals/README.md]: Newlines for list?

Comment (smit1678) [reply quarterly-goals/README.md]: 👍

**Comment by awright** [comment]
agree with @bgirardot -- all sounds good here to me too.

**Comment by smit1678** [comment]
Great thanks. Will merge and can open up any further discussions to make adjustments or improvements. 🎉

**Pull request merged**